core/src/main/scala/clue/ErrorPolicy.scala

```diff
@@ -31,15 +32,24 @@ object ErrorPolicy {
       ApplicativeThrow[F].raiseError(ResponseException(errors, data))
   }
 
+  /**
+   * If the response contains data, return it. If the response contains errors, raise an exception.
+   * If the response contains both data and errors, log the errors and return the data.
+   */
   object IgnoreOnData extends ErrorPolicy {
     type ReturnType[D] = D
 
     def processor[D]: ErrorPolicyProcessor[D, D] = new Distinct[D] {
-      def process[F[_]: ApplicativeThrow](response: GraphQLResponse[D]): F[D] =
+      def process[F[_]: ApplicativeThrow](
+        response: GraphQLResponse[D]
+      )(implicit log: Logger[F]): F[D] =
         response.result match {
-          case Ior.Left(errors)  => processErrors(errors)
-          case Ior.Right(data)   => processData(data)
-          case Ior.Both(_, data) => processData(data)
+          case Ior.Left(errors)       => processErrors(errors)
+          case Ior.Right(data)        => processData(data)
+          case Ior.Both(errors, data) =>
+            log.error(ResponseException(errors, data.some))(
```

Comment:
Not a big issue, but maybe we want to log as `warn` or `info`, since we are explicitly choosing to ignore the error in this case.

Reply:
Good point, changed it to warn